vocabularies/gsq-roles.ttl

```diff
@@ -0,0 +1,80 @@
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix gsqr: <http://linked.data.gov.au/def/gsq-roles/> .
+@prefix iso11179: <http://linked.data.gov.au/def/iso11179-6/RolesAndResponsibilities/> .
+@prefix isorole: <http://linked.data.gov.au/def/iso19115-1/RoleCode/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+
+<http://linked.data.gov.au/def/gsq-roles> a owl:Ontology .
+
+gsqr:conceptScheme a skos:ConceptScheme ;
+    dct:source <https://standards.iso.org/iso/19115/resources/Codelists/gml/CI_RoleCode.xml> ;
+    skos:definition "The roles of Agents (People and Organisations) in relation to Entities (Datasets, Samples etc.)"@en ;
+    skos:hasTopConcept gsqr:contractor,
+        iso11179:Submitter,
+        isorole:author,
+        isorole:owner,
+        isorole:publisher ;
+    skos:prefLabel "Geological Survey of Queensland's Agent Roles"@en .
+
+gsqr:QdexReportsRoles a skos:Collection ;
+    skos:definition "A collection of roles used for Agents within the QDEX Reports regime"@en ;
+    skos:member gsqr:contractor,
+        gsqr:driller,
+        gsqr:operator,
+        iso11179:Submitter,
+        isorole:author,
+        isorole:owner,
+        isorole:publisher ;
+    skos:prefLabel "Qdex Reports Roles"@en .
+
+gsqr:driller a skos:Concept ;
+    skos:broader gsqr:contractor ;
+    skos:definition "Driller"@en ;
+    skos:inScheme isorole:conceptScheme ;
+    skos:prefLabel "Contractors who drill bores/wellbores/boreholes etc."@en ;
+    skos:topConceptOf gsqr:conceptScheme .
+
+gsqr:operator a skos:Concept ;
+    skos:broader gsqr:contractor ;
+    skos:definition "Operator"@en ;
+    skos:inScheme isorole:conceptScheme ;
+    skos:prefLabel "Contractors who operate bores/wellbores/boreholes etc., mines, quarries etc."@en ;
+    skos:topConceptOf gsqr:conceptScheme .
```

Comment:
From P&G (general provisions) regulation 2017
"the corporation or other entity that is the decision-maker and has responsibility for the overall management for the project."

Unless there is a specific need (such as misapplication of terms, or poor definitions), I suggest we just take definitions directly from our Legislation and Regulation where they exist.

Reply:
Addressed as recommended by commit https://github.com/geological-survey-of-queensland/vocabularies/pull/60/commits/9e940a840009645c08f7eb93f3a2cc3397d78bd9